zerver/models.py

```diff
@@ -1116,6 +1123,10 @@ def is_realm_admin(self, value: bool) -> None:
             # ROLE_GUEST to ROLE_MEMBER here.
             self.role = UserProfile.ROLE_MEMBER
 
+    @property
+    def is_realm_owner(self) -> bool:
+        return self.role == UserProfile.ROLE_REALM_OWNER
+
     @property
```

Comment:
I was not sure whether we want to add setter for this as it was not required till now.

Reply:
Yeah, we can probably skip the setter for this.